### DISCUSSION TITLE
Get results from a specific website with "site:" operator

### DISCUSSION THREAD

**Discussion by hsurden** [qunash/chatgpt-advanced#54] [Ideas]
It would be great if we could get our web results from a particular website, with the "site:" functionality.  For instance, if we wanted to search "Google Scholar", maybe we could preface our query with "site:scholar.google.com" and the search would bring in results from that site only, or from WebMd ("site:webmd.com").   

This would be similar to the way if you do a search in google using the "site:reddit.com chatgpt", Google will give you only the results from reddit.com that match chatgpt.  Thank you

**Comment by qunash** [comment]
Hi, "site:" operator is already supported.
